server/handlers/handlers.go

```diff
@@ -47,6 +47,7 @@ const (
 	discardedReasonMaxFileSizeExceeded    = "max_file_size_exceeded"
 
 	actionSourceMSTeams    = "msteams"
+	ActionSourceMattermost = "mattermost"
```

Comment:
This doesn't feel like a constant that `handlers` should define. Can we export this from `metrics` instead?